VogtPayroll2/Program.cs

```diff
@@ -1,12 +1,102 @@
 ﻿using System;
+using System.Collections.Generic;
 
 namespace VogtPayroll2
 {
     class Program
     {
+        private static int employeeCounter;
```

Comment:
You can get rid of the employeeCounter.

There are two options for this:

# Option 1

This option is better if you do not have a reason to already fetch all of the employees from the PayrollManager.

Create a function in `PayrollManager` that returns the total count:
```c#
public int GetEmployeeCount() {
  return _employees.Count;
}
```

Then reference it anywhere you need it:
```c#
  payrollManager.GetEmployeeCount();
```

# Option 2

If you have a reason to fetch all of the employees from the payrollManager, you could add this instead to the `PayrollManager` class:
```c#
public List<Employee> GetEmployees() {
  return _employees
}
```

Then anywhere you need to get the total list of employees, you can do:
```c#
  payrollManager.GetEmployees().Count;
```